src/api/verify.ts

```diff
@@ -78,14 +83,86 @@ class VerifyAPI {
       expiresAt,
       expiresIn,
       passed,
+      failureReasons,
+      _id,
     } as RadarTrackVerifiedResponse;
 
     if (options.debug) {
       trackRes.response = response;
     }
 
+    lastToken = trackRes;
+    lastTokenNow = performance.now();
```

Comment:
We should use `Date.now()` unless there's a need for nano-second measurement (this is really only intended for perf measurement)

Reply:
The problem with `Date.now()` is that you can manipulate it by changing your system clock time, whereas `performance.now()` is based off of time since page load and harder to tamper with

Reply:
We do something similar on the mobile SDKs (`System.elapsedRealTimeNanos` on Android) for the same reason